src/sentry/static/sentry/app/views/projectInstall/overview.jsx

```diff
@@ -13,10 +13,11 @@ const ProjectInstallOverview = React.createClass({
 
   getIntegrationLink(root, platform, display) {
     let {orgId, projectId} = this.props.params;
+    let onboarding = this.props.location.query.onboarding ? 'onboarding=1' : '';
     return (
       <li className={`${root} ${platform}`} key={platform}>
         <span className={`platformicon platformicon-${platform}`}/>
-        <Link to={`/${orgId}/${projectId}/settings/install/${platform}/`}>
+        <Link to={`/${orgId}/${projectId}/settings/install/${platform}/?${onboarding}`}>
```

Comment:
Done, though I'm doing it in a bit of a hack-y way — setting onboarding to equal "?onboarding=1". Since we're not using params otherwise, seems fine?

This honestly is a nice-to-have, to be able to say "first event pending" when we *think* they've installed the SDK. Will be a lot better once we have a VCS integration and we can for sure see it in the code.

cc @benvinegar